src/main/java/com/dougnoel/sentinel/elements/tables/Table.java

```diff
@@ -398,8 +398,10 @@ public void clickElementInRowThatContains(int ordinalRow, By elementLocator) {
 	 * @return ArrayList&lt;String&gt; a list of all cell data. Each entry in the list corresponds to a cell.
 	 */
 	public List<String> getAllCellDataForColumn(String columnHeader){
-		getOrCreateHeaders();
-		return getOrCreateColumns().get(columnHeader);
+		int indexOfHeader = getOrCreateHeaders().indexOf(columnHeader) + 1;
```

Comment:
I don't love comments typically in code, but in this case I'd actually like you to add a comment about why the +1 is being added. This is one of those things that is very confusing coming into it.